dev/benchmarks/macrobenchmarks/pubspec.yaml

```diff
@@ -17,6 +17,18 @@ dependencies:
   #   flutter update-packages --force-upgrade
   flutter_gallery_assets: 0.2.0
 
+  flutter_gallery_additional_assets:
+    git:
+      url: https://github.com/pennzht/fluttergalleryadditionalassets.git
+      ref: f4a7c99a56df62496bc057adb9e7b5b916e0f451
+
+  # This is needed for web_benchmarks.
+  # After the gallery PR merges, change this to a specific commit on flutter_gallery:master.
+  gallery:
+    git:
+      url: git://github.com/pennzht/newfluttergallery.git
+      ref: 0fef0f90df5ddf49b7822933dadf43b2da7fb025
```

Comment:
The urls will need to be updated before merging.

Reply:
Added a TODO.